frontend/src/features/coverLetter/components/CoverLetter.tsx

```
@@ -136,56 +124,79 @@ const CoverLetter = ({
+          role='presentation'
+          style={{
+            top: selection.modalTop + SPACER_HEIGHT,
+            left: selection.modalLeft,
+          }}
+          onMouseDown={(e) => {
+            e.stopPropagation();
+          }}
+        >
+          <ReviewModal
+            review={editingReview}
+            initialRevision={editingReview?.revision}
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)